src/app/pages/OnDemandAudioPage/OnDemandAudioPage.jsx

```diff
@@ -49,20 +51,30 @@ const StyledGelWrapperGrid = styled(GelPageGrid)`
   @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
     padding-top: ${GEL_SPACING_TRPL};
   }
+  @media screen and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+    display: flex;
+    flex-direction: row;
+  }
 `;
 
 const StyledGridItemParagraph = styled(Grid)`
-  @media (min-width: 22.5rem) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+  @media (min-width: ${GEL_GROUP_1_SCREEN_WIDTH_MIN} and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
     grid-template-columns: repeat(4, 1fr);
     grid-column-end: span 4;
   }
 `;
 
 const StyledGridItemImage = styled(Grid)`
-  @media (min-width: 22.5rem) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+  @media (min-width: ${GEL_GROUP_1_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
     grid-template-columns: repeat(2, 1fr);
     grid-column-end: span 2;
   }
+  @media screen and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+    flex-basis: 33.33%;
+  }
+  @media (max-width: ${GEL_GROUP_1_SCREEN_WIDTH_MAX}) {
+    display: none;
+  }
```

Comment:
Also is this behaviour being added or does it already exist?

Reply:
This behaviour is specific for IE only but I will give this another go to fix the underlying IE issue. Simorgh is not falling back to the default grid